django_prices_vatlayer/models.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import pgettext_lazy
+
+
+@python_2_unicode_compatible
+class EuropeanVatRate(models.Model):
```

Comment:
Why not rename this to `VAT`? We don't explicitly filter the API to only ask for European VAT rates so it's theoretically possible for a non-European country to join the VAT union and automatically appear in the list. It's also not a rate as multiple rates are saved for each country.